pom.xml

```diff
@@ -48,33 +59,71 @@
             <url>https://oss.sonatype.org/service/local/staging/deploy/maven2/</url>
         </repository>
     </distributionManagement>
-	
-	<properties>
-		<version.junit>4.11</version.junit>
-        <logback.version>1.0.13</logback.version>
-	</properties>
-	
-	<modules>
-		<module>transport</module>
-		<module>cluster</module>
-	</modules>
-	
-    <dependencies>
-		<dependency>
-			<groupId>junit</groupId>
-			<artifactId>junit</artifactId>
-			<version>${version.junit}</version>
-			<scope>test</scope>
-		</dependency>
-        <dependency>
-            <groupId>ch.qos.logback</groupId>
-            <artifactId>logback-classic</artifactId>
-            <version>${logback.version}</version>
-            <scope>test</scope>
-        </dependency>
-    </dependencies>
+
+    <properties>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
+
+        <slf4j.version>1.7.7</slf4j.version>
+        <rxjava.version>1.0.6</rxjava.version>
+        <guava.version>18.0</guava.version>
+        <findbugs.version>2.0.3</findbugs.version>
+        <netty.version>4.0.28.Final</netty.version>
+        <protostuff.version>1.3.2</protostuff.version>
+        <protobuf.version>2.4.1</protobuf.version>
+    </properties>
+
+    <modules>
+        <module>transport</module>
+        <module>cluster</module>
+        <module>testlib</module>
+    </modules>
+
+    <dependencyManagement>
+        <dependencies>
+            <dependency>
+                <groupId>org.slf4j</groupId>
+                <artifactId>slf4j-api</artifactId>
+                <version>${slf4j.version}</version>
+            </dependency>
+            <dependency>
+                <groupId>com.google.guava</groupId>
+                <artifactId>guava</artifactId>
+                <version>${guava.version}</version>
+            </dependency>
+            <dependency>
+                <groupId>io.reactivex</groupId>
+                <artifactId>rxjava</artifactId>
+                <version>${rxjava.version}</version>
+            </dependency>
+            <dependency>
+                <groupId>com.google.code.findbugs</groupId>
+                <artifactId>jsr305</artifactId>
+                <version>${findbugs.version}</version>
```

Comment:
Lets put here to dependency management all dependencies we use in project and in each module their dependencies (without versions)